src/lib/components/DatePickerPanel.js

```diff
@@ -7,6 +7,52 @@ import Calendar from 'rc-calendar';
 import TimePicker from 'rc-time-picker';
 import { use12hFormat } from '../time-formats.js';
 
+import cs_CZ from 'rc-calendar/lib/locale/cs_CZ';
+import da_DK from 'rc-calendar/lib/locale/da_DK';
+import de_DE from 'rc-calendar/lib/locale/de_DE';
+import en_US from 'rc-calendar/lib/locale/en_US';
+import ja_JP from 'rc-calendar/lib/locale/ja_JP';
+import pl_PL from 'rc-calendar/lib/locale/pl_PL';
+import pt_BR from 'rc-calendar/lib/locale/pt_BR';
+import ru_RU from 'rc-calendar/lib/locale/ru_RU';
+import zh_CN from 'rc-calendar/lib/locale/zh_CN';
```

Comment:
Ugh, yeah... Rollup only does static imports, because it does dependency analysis. Browserify (and I think Webkit) could do this dynamically, but meh.